README.md

```diff
@@ -1,40 +1,119 @@
 # shield-study-rappor
 
-## install
+## Introduction
+This addon is a SHIELD Study that collects the eTLD+1 of the user's homepage
+and applies [RAPPOR](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/42852.pdf),
+a [Differential Privacy](https://en.wikipedia.org/wiki/Differential_privacy) algorithm, to encode it.
+The result is sent in a custom ping.
+
+The purpose of this study is to answer some questions with unbiased data such as:
+
+- "Which top sites are users visiting?"
+- "Which sites using Flash does a user encounter?"
+- "Which sites does a user see heavy Jank on?" 
+
+## Addon Behaviour
+The addon collects the user's homepage from the `browser.startup.homepage` preference.
```

Comment:
Maybe be clearer on this: "The addon extracts the eTLD+1 from the value stored in the `browser.startup.homepage`  preference and applies RAPPOR to make it anonymous. It can fail if ... . For example, if the value stored in the preference is `foo.bar.com` the addon applies RAPPOR to `bar.com` and then sends the anonymized bit field out."

Reply:
Done